.dockerignore

```diff
@@ -0,0 +1,18 @@
+.vagrant
+MYMETA.*
+Makefile
+blib
+local
+pm_to_blib
+template_cache
+vagrant_support
+\#*\#
+*/\#*\#
+*/*/\#*\#
+*/*/*/\#*\#
+*/*/*/*/\#*\#
+.DS_Store
+*/.DS_Store
```

Comment:
Docker technically is trying to support ```**/.DS_Store```, but they have 3 implementation bugs right now around it, and docker-compose wasn't handling it right in January, so this is fine, if aesthetically grumpy.